docs/user/integration/aws-cloudwatch/README.md

```diff
@@ -0,0 +1,152 @@
+
+
+## Preparation
+
+1. Export your namespace as a variable. Replace the `{NAMESPACE}` placeholder in the following command and run it:
+
+    ```bash
+    export K8S_NAMESPACE="{NAMESPACE}"
+    ```
+1. If you haven't created a namespace yet, do it now:
+    ```bash
+    kubectl create namespace $K8S_NAMESPACE
+    ```
+## Set Up AWS Credentials
+
+### Create AWS IAM User
+
+1. In your AWS account, create an [IAM policy](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies.html) for the **CloudWatch Logs** service with the actions **CreateLogGroup**, **CreateLogStream**, **PutLogEvents**, and **PutRetentionPolicy**, and specify the resource [ARNs](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference-arns.html) for the selected actions.
+2. In your AWS account, create an [IAM user](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users.html) and attach the policy you just created, as well as the policy **AWSXrayWriteOnlyAccess**.
+3. For the [IAM user](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users.html) you just created, create an access key for an application running outside AWS. Copy and Save the **access key** and **secret access key** as you will need them in the next step [Create a Secret with AWS Credentials](#Create-a-Secret-with-AWS-Credentials)
+
+
+### Create a Secret with AWS Credentials
+
+To connect the AWS Distro to the AWS services, create a secret containing the credentials of the created IAM user into the Kyma cluster. In the following command, replace `{ACCESS_KEY}` with your access key, `{SECRET_ACCESS_KEY}` with your secret access key, and `{AWS_REGION}` with the AWS region you want to use:
+ 
+```bash
+kubectl create secret generic aws-credentials -n $K8S_NAMESPACE --from-literal=AWS_ACCESS_KEY_ID={ACCESS_KEY} --from-literal=AWS_SECRET_ACCESS_KEY={SECRET_ACCESS_KEY} --from-literal=AWS_REGION={AWS_REGION}
+ ```
+
+## Deploy the AWS Distro
+
+Deploy the AWS Distro, which is an AWS-supported distribution of an OTel collector. The [AWS X-Ray Tracing Exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/awsxrayexporter) used in the collector converts OTLP traces to [AWS X-Ray Segment Documents](https://docs.aws.amazon.com/xray/latest/devguide/xray-api-segmentdocuments.html) and then sends them directly to X-Ray. The [AWS CloudWatch EMF Exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/awsemfexporter/README.md) used in the collector converts OTLP metrics to [AWS CloudWatch Embedded Metric Format(EMF)](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Embedded_Metric_Format_Specification.html) and then sends them directly to CloudWatch Logs. The retention of these CloudWatch Logs is set to 7 days. You can change that to fit your needs by adjusting the `log_retention` value for the `awsemf` exporter in the [`aws-otel.yaml`](aws-otel.yaml) file.
```

Comment:
```suggestion
Deploy the AWS Distro, which is an AWS-supported distribution of an OTel collector. The [AWS X-Ray Tracing Exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/awsxrayexporter) used in the collector converts OTLP traces to [AWS X-Ray Segment Documents](https://docs.aws.amazon.com/xray/latest/devguide/xray-api-segmentdocuments.html) and then sends them directly to X-Ray. The [AWS CloudWatch EMF Exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/awsemfexporter/README.md) used in the collector converts OTLP metrics to [AWS CloudWatch Embedded Metric Format(EMF)](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Embedded_Metric_Format_Specification.html) and then sends them directly to CloudWatch Logs.

> **NOTE:** The retention of these CloudWatch Logs is set to 7 days. You can change that to fit your needs by adjusting the `log_retention` value for the `awsemf` exporter in the [`aws-otel.yaml`](aws-otel.yaml) file.
```

Why mention adjusting the retention value twice, here and in line 67? Can we drop it here? If not, I'd at least separate it as a new paragraph and maybe as Note.

Reply:
They are not the same.

Here, it is the retention value for the CloudWatch Logs which are sent by the AWS Distro. These logs are originally OTLP metrics, but they are converted by AWS Distro to AWS CloudWatch Embedded Metric Format and then sent to CloudWatch Logs.

While the other retention value which is set in the custom output of the Logpipeline is for the CloudWatch Logs which are sent by Fluent Bit. These logs are originally OTLP logs which are converted by the Amazon CloudWatch output plugin to a format accepted by CloudWatch and then sent to CloudWatch.